cltools.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
This whole thing makes me a little nervous.  I really like the direction of trying to make it easy to acquire the toolchain, but I worry about doing all this magic behind the scenes (especially the need to sudo to put stuff in /usr/lib).  Do you know if we have to do this (i.e. are the inbox versions of autconf, automake and libtool installed as part of the OSX dev tools)?  If we must use external tools, could we instead recommend that folks use `brew` to acquire them or something, similar to the guidance we've given in the past for cmake?

I'd much rather us just document what the toolchain is and leave the acquisition up to the end developer.